tools/scripts/show-runtime-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Quick script to output the runtime spec_version at a particular commit.
```

Comment:
Helpful for getting the spec_version in the past. I can drop this file if people think it isn't worth it.

Reply:
💯